bynder_sdk/util.py

```diff
@@ -0,0 +1,33 @@
+def api_endpoint_url(session, endpoint):
+    return 'https://{}/api{}'.format(session.bynder_domain, endpoint)
+
+
+def parse_json_for_response(response):
+    try:
+        return response.json()
+    except ValueError:
+        return None
+
+
+class SessionMixin:
```

Comment:
sorry for the annoyance, but why not calling this class `Session`? I find the `SessionMixin` name a bit confusing

Reply:
Well because they're not the true parent of the classes that "inherit" from it, it's just an addition that we use on top of the actual parent class. For example, the declaration of the `BynderAPISession` class is:

```python
class BynderAPISession(SessionMixin, Session):
```

In python the inheritance goes from right to left, so it builds it like this: `Session -> SessionMixin -> BynderAPISession`. I should probably rename `BynderAPISession` to `PermanentTokenSession` for consistency with the `OAuth2Session`, what do you think?

Reply:
Yes that would make it more understandable.

Also my bad for not knowing the term `Mixin`.